packages/11ty/_includes/components/figure/table/element.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+
+/**
+ * Renders a table from a static html file referenced in a figure
+ *
+ * @param      {Object}  eleventyConfig  eleventy configuration
+ * @param      {Object}  figure          The figure object
+ *
+ * @return     {String}  Text content of referenced file
+ */
+module.exports = function (eleventyConfig) {
+  const { imageDir } = eleventyConfig.globalData.config.params
+  const assetsDir = path.join(eleventyConfig.dir.input, imageDir)
```

Comment:
_In a separate pull-request_ we might make `assetsDir` a computed global data property.